src/main/modules/routes/routes.ts

```diff
@@ -251,6 +252,8 @@ export class Routes {
     app.get(PageUrls.APPLICATION_DETAILS, new ApplicationDetailsController().get);
     app.get(PageUrls.RESPOND_TO_TRIBUNAL, new RespondToTribunalController().get);
     app.post(PageUrls.RESPOND_TO_TRIBUNAL, new RespondToTribunalController().post);
+    app.get(PageUrls.RETURN_TO_EXISTING_RESPONSE, new ReturnToExistingResponseController().get);
```

Comment:
Is it possible to move this above or below, so that it would not split the RESPOND_TO_TRIBUNAL? just for easy reading